databricks/koalas/indexes.py

```diff
@@ -456,6 +457,47 @@ def to_series(self, name: Union[str, Tuple[str, ...]] = None) -> Series:
                                          column_index_names=None),
                       anchor=kdf)
 
+    def duplicated(self, keep=False):
```

Comment:
I think we can just explicitly unsupport this API as its return type is NumPy arrow and the data size seems expected to be large.
We can just say users can easily work around by `DataFrame.deduplicated()` API.

```python
>>> kdf.index.to_series().to_frame().duplicated()
0    False
1    False
2    False
3    False
4    False
5    False
6    False
7    False
8    False
9    False
Name: 0, dtype: bool
```

I think we shouldn't implement `Series.duplicated()` as well. 

Can you refer https://github.com/databricks/koalas/pull/744 and fix it accordingly?

Reply:
oh wait.
Seems like `Series.duplicated()` returns `Series`. We can support it.
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.duplicated.html

Reply:
Yeah, I missed it. @RainFung, can you implement this one?

Reply:
maybe we can implement it in a separate PR. just revert the updates for Series missing file.

Reply:
OK，  i'll fix it today